packages/db/src/migrations/20240614123146_add_member_id_admin.ts

```diff
@@ -0,0 +1,24 @@
+import { type Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  await db.schema
+    .alterTable('admins')
+    .addColumn('member_id', 'text', (column) => {
+      return column.references('students.id').onDelete('cascade');
+    })
+    .execute();
+
+  await db
+    .updateTable('admins')
+    .set({
+      member_id: db
+        .selectFrom('students')
+        .select(['id'])
+        .where('students.email', '=', 'admins.email'),
```

Comment:
Here we want to use `.whereRef` rather than `.where`

Reply:
fixed!